util/chplenv/utils.py

```diff
@@ -19,15 +25,15 @@ def memoize_wrapper(*args, **kwargs):
 
 @memoize
 def get_chpl_home():
-    chpl_home = os.environ.get('CHPL_HOME', '')
+    chpl_home = defaults.get('CHPL_HOME', '')
     if not chpl_home:
         dirname = os.path.dirname
         chpl_home = dirname(dirname(dirname(os.path.realpath(__file__))))
     return chpl_home
 
 @memoize
 def using_chapel_module():
-    chpl_home = os.environ.get('CHPL_HOME', '')
+    chpl_home = defaults.get('CHPL_HOME', '')
     if chpl_home != '':
         return chpl_home == os.environ.get('CHPL_MODULE_HOME', '')
```

Comment:
I purposefully did not include CHPL_MODULE_HOME since it is not one of the 23 env vars printed with `printchplenv --simple`. We could add it to the list though.